examples/restapi/README.md

```diff
@@ -6,11 +6,31 @@ Using GraphQL it's possible to write resolvers that call some other APIs.
 const resolvers = {
   Query: {
     queryREST: async (obj, args, context) => {
-       const result = await axios.get(YOUR_ENDPOINT_HERE, params);
-       return result.data;
+      try {
+        const result = await axios.get(YOUR_ENDPOINT_HERE, params);
+        return result.data;
+      } catch (e) {
+        throw new Error("some error");
```

Comment:
GraphQLError ?

Reply:
I didn't mention any difference here, but ok.

Reply:
If we provide info how we handle errors we should do it the way that GraphQL recommends that. 
It's small thing but it will make huge difference.